public/js/clipboard.min.js

```diff
@@ -1,7 +0,0 @@
-/*!
```

Comment:
Please be sure we aren't using this.  I though we had clipboard functionality in use somewhere.

Reply:
We had it a while ago, don't anymore. If we need need to add it again, can probably do that without use of this library.